.github/workflows/release.yml

```diff
@@ -16,7 +16,6 @@ jobs:
         ref: master
     - uses: actions/setup-node@v3
       with:
-        node-version: 14
```

Comment:
this is a CI script that use `ubuntu-latest` Docker image, so this way, it will use the latest stable Node which should be always supported by Fomantic-UI

Reply:
The latest stable node does not necessarily mean that the nightly script will still work in any future version! If this isnt tested for node 22 (for example) and the docker images gets updated and, by whatever (unlikely) incompatibility the scripts does not work (or throws any unexpected error) it will break
Yes, unlikely it breaks, but who knows. As said, even setup-node suggests to state an explicit version 

Remember, we had the situation where Node 15/16 was previouly not working with fomantic anymore because of an incompatibility with inquirere not supporting yarn with interactive scripts . We had this before, that's why :wink:

Yes, Fomantic _should_ support the latest node version, but only until tested

Reply:
I prefer no fixed numbers anywhere so I added `latest` to the CI.

- the release script (outside the push to registry) is the same as our CI test
- the nighly script is the same as well plus one file to compute the next version

so release script should never fail and nighly script can fail, but I would say nighly failure is good, as we can detect the problems :)